packages/common/amplify_db_common/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: amplify_db_common
+description: Common utilities for working with databases such as sqlite.
+version: 0.1.0
+homepage: https://github.com/aws-amplify/amplify-flutter/tree/next
+repository: https://github.com/aws-amplify/amplify-flutter/tree/next/packages/common/amplify_db_common
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+
+dependencies:
+  amplify_core: ">=1.0.0-next.0+1 <1.0.0-next.1"
+  async: ^2.8.0
+  drift: ^2.2.0
+  meta: ^1.7.0
+  path: ^1.8.0
+  sqlite3: ^1.9.0
```

Comment:
Did you test running this on Android and iOS?  I wanted to just use sqlite3 dep as well, but couldn't and had to use sqlite3_flutter_libs ...  Actually had a big issue with this on Android with the exception because of some missing sqlite.so file.  

Furthermore, the Drift docs say sqlite3_flutter_libs are required for it to work on Linux/Windows unless an sqlite3.so is included in the app.

Maybe we could import sqlite3_flutter_libs for Analytics and Storage Flutter portion, and possibly keep the import of sqlite3 here.  Would need to test that doesn't lead to duplication issues though.

Reply:
`sqlite3_flutter_libs` is a Flutter-only package, so it should be a dependency in the Flutter-only category packages. For Windows/Linux, all it does is bundle it in like you mention. This library is meant to be a Dart-only package with the loading business logic of the assumed already-bundled library.

Reply:
Nice thank you for the detailed explanation.  All sounds good.

Reply:
Cool, yeah that sounds good. I agree we should do it in the future.